packages/yoastseo/src/languageProcessing/languages/fr/helpers/checkIfWordIsComplex.js

```diff
@@ -45,16 +46,19 @@ export default function checkIfWordIsComplex( config, word ) {
 	if ( word[ 0 ].toLowerCase() === word[ 0 ] ) {
 		/*
 		 * If a word is longer than 9 characters and doesn't start with capital letter,
-		 * we check further whether it is a plural ending in -s. If it is, we remove the -s suffix
+		 * we check further whether it is a plural that ends on the -s or -x plural suffix. If it is, we remove the plural suffix
 		 * and check if the singular word can be found in the frequency list.
+		 * if it is a plural that does not end on -s or -x but on -aux, we replace the plural -aux suffix with the singular suffix -al.
 		 * The word is not complex if the singular form is in the list.
 		 */
-		if ( word.endsWith( "s" ) ) {
+		if ( word.test( morphologyData.suffixGroupsComplexity.standardSuffixesWithSplural ||
+			morphologyData.suffixGroupsComplexity.standardSuffixesWithXplural ) ) {
 			word = word.substring( 0, word.length - 1 );
-			return ! frequencyList.includes( word );
+		} else if ( word.endsWith( morphologyData.suffixGroupsComplexity.irregularPluralSingularSuffixes ) ) {
+			word = word.replace( new RegExp( morphologyData.suffixGroupsComplexity.irregularPluralSingularSuffixes[ 0 ] ),
+				morphologyData.suffixGroupsComplexity.irregularPluralSingularSuffixes[ 1 ] );
 		}
-		return true;
+		return ! frequencyList.includes( word );
 	}
-
-	return false;
+	return true;
```

Comment:
What was the reason this was changed to `return true;` instead of `return false;`? 🤔 
Because then if this is changed to return `true`, words longer than 9 characters that start with capital letter is going to be considered complex. And that's not the case in French right?